extensions/resteasy-classic/resteasy-jaxb/deployment/src/test/java/io/quarkus/resteasy/jaxb/deployment/ConsumesXMLTestCase.java

```diff
@@ -25,8 +25,8 @@
     @Test
     public void testConsumesXML() {
         RestAssured.given()
-                .body(new Bar("open", "bar"))
```

Comment:
No idea how this ever worked, the body() call does the serialization, so you need to set the content type first.